### DISCUSSION TITLE
добавления

### DISCUSSION THREAD

**Discussion by JellyfishKa** [Flowseal/zapret-discord-youtube#896] [Ideas]
Доброго времени суток! Можете ли добавить поддержку основных аниме-сайтов, пожалуйста?

**Comment by testidcode** [comment]
1) "основные" это какие? abema, animejoy..? У каждого свое
2) В релиз это никто не добавит, тк это не проблема большинства, если вам нужны - добавьте адреса нужных сайтов в list-general.txt и перезапустите bat/службу/комп. 
К тому же это не имеет огромного смысла, тк владельцам сайтов самим выгодно, чтобы трафик был, поэтому через день-два появляются зеркала, зачастую с авто-редиректом со старого домена

**Comment by MohBarakat** [comment]
Can you add ChatGPT so we can access it, please?

**Reply by testidcode** [reply]
Openai themselves blocked access to users from the Russian Federation, Belarus, etc. And determines the user's country by IP. The program does not change your IP, so you need to use a VPN or proxy

**Comment by markorlov02** [comment]
предлагаю сделать обход блокировки инстаграм

**Comment by RGH686** [comment]
If anyone is interested, I'm selling my username. On the site Fragment.com
DiscordYoutube 
YouTube_Discord
In Telegram

**Comment by reikidod** [comment]
Добавить бат файл который будет по очереди запускать все general.bat и тестировать на работоспособность и выбирать самые подходящие ну или же рабочие
